src/web/components/Figure.tsx

```diff
@@ -7,7 +7,7 @@ import { space } from '@guardian/src-foundations';
 type Props = {
 	children: React.ReactNode;
 	isMainMedia?: boolean;
-	role?: RoleType;
+	role?: RoleType | 'richLink';
```

Comment:
curious to know why `richLink` isnt a Role?

Reply:
I wanted to be explicit that `richLink` was being added as an exception here to handle the fact that rich links are a bit special. If I were to 'officially' add a new `role` type to the model then that would involve a lot more work to add support and testing for it plus it would no longer match the data being sent from CAPI.

It's not ideal, but based on our discussions in the chat channel, this seemed the best approach.